### PR TITLE
add kubeedge web demo

### PR DESCRIPTION
**Pull request by edisonxiang**
add kubeedge web demo

### CODE REVIEW

File: kubeedge-web-demo/Gopkg.toml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+#   unused-packages = true
+
+
+[[constraint]]
+  name = "github.com/astaxie/beego"
+  version = "1.11.1"
+
+[[constraint]]
+  name = "github.com/kubeedge/kubeedge"
+  version = "0.3.0"
```
Comment (fisherxu) [inline kubeedge-web-demo/Gopkg.toml]: please help bump to 1.0.0 :)

File: kubeedge-web-demo/README.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+* RaspBerry PI (RaspBerry PI 3 has been used for this demo).
+  The RaspBerry PI is also the edge node to which the speaker will be connected.
+
+* A speaker for playing the music.
+
+### Software Prerequisites
+
+* A running Kubernetes cluster.
+
+* KubeEdge [0.3.0](https://github.com/kubeedge/kubeedge/releases/tag/v0.3.0)
```
Comment (fisherxu) [inline kubeedge-web-demo/README.md]: ditto

File: kubeedge-web-demo/Gopkg.toml (hunk 1 above)
Comment (edisonxiang) [reply kubeedge-web-demo/Gopkg.toml]: Done

File: kubeedge-web-demo/README.md (hunk 2 above)
Comment (edisonxiang) [reply kubeedge-web-demo/README.md]: Done

**Comment by edisonxiang** [comment]
Thanks @fisherxu 
New commit is added:)

**Comment by fisherxu** [comment]
LGTM

File: kubeedge-web-demo/README.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
+# KubeEdge Web Demo
+
+Firstly the users input `kubeedge` to get the web app link,
```
Comment (rohitsardesai83) [inline kubeedge-web-demo/README.md]: where do they need to provide this input ? in a browser, app ?

Comment (edisonxiang) [reply kubeedge-web-demo/README.md]: Updated. Thanks @rohitsardesai83

**Comment by edisonxiang** [comment]
cc @sids-b

**Pull request merged**